### ISSUE TITLE
Missing typings

### ISSUE THREAD

**Issue by pooledge** [verlok/vanilla-lazyload#555]
```
/**
 * The class applied to the elements after they entered the viewport.
 *
 * @default "entered"
 */
class_entered?: string;

/**
 * The class applied to the elements after they exited the viewport.
 *
 * @default "exited"
 */
class_exited?: string;
```

Stumbled upon these two, at least.
Sorry @verlok I'm on the road, cannot make a PR :[

**Comment by verlok** [comment]
Hey @pooledge 
thank you for reporting. 

Would you be able to make a PR now?

**Comment by verlok** [comment]
Fixed in 17.5.1

Thanks for reporting @pooledge

**Comment by verlok** [comment]
Hey @pooledge  
thank you again for reporting this. 

If you’re happy with my support, the documentation and the effort I’ve been putting on this project in the latest years, I hope you might want to [buy me a coffee](https://ko-fi.com/verlok) to show your appreciation. Or [sponsor me](https://github.com/sponsors/verlok), if you're a fan.

Open-source software is great for everyone, but it takes passion and time (and coffee!) to grow and evolve.

Thank you for thinking about it. 
Have a wonderful day!
